### PR TITLE
Rename dumptsmdev to dumptsm in influx_inspect

### PR DESCRIPTION
**Pull request by jsternberg**
Removes the old implementation of `dumptsm`. It was for an older version
of the tsm1 files that is no longer used and now just panics when used
on a tsm1 file. dumptsmdev has been renamed to `dumptsm`, but the old
`dumptsmdev` command still works for compatibility.

### CODE REVIEW

File: cmd/influx_inspect/main.go [hunk 1]
```diff
@@ -49,7 +48,7 @@ func main() {
 			os.Exit(1)
 		}
 		cmdInfo(path)
-	case "dumptsm":
+	case "dumptsm", "dumptsmdev":
```
Comment (jwilder) [inline cmd/influx_inspect/main.go]: Can you log a warning that `dumptsmdev` is deprecated and to use `dumptsm` going forward?

**Comment by jwilder** [comment]
I think `readFields` and `readIndex` are now dead code too and can be removed.

File: cmd/influx_inspect/main.go (hunk 1 above)
Comment (jsternberg) [inline cmd/influx_inspect/main.go]: Done.

**Comment by jsternberg** [comment]
Yep. I removed all of the stuff `deadcode` told me about (running iteratively after I removed each thing it reported).

**Comment by jwilder** [comment]
👍 on green.

**Pull request merged**